moped-editor/src/queries/project.js

```diff
@@ -250,7 +250,7 @@ export const TIMELINE_QUERY = gql`
       phase_id
       phase_name
       phase_order
-      moped_subphases(order_by: { subphase_order: asc }) {
```

Comment:
@chiaberry would you sort this query and the ones below by `subphase_name` instead? that way the dropdown menu's will render the options consistently.